rules/S7155/secrets/metadata.json

```
@@ -0,0 +1,56 @@
+{
+  "title": "CircleCI secrets should not be disclosed",
```

```suggestion
  "title": "CircleCI API tokens should not be disclosed",
```

I would pick one word for the name of the secret and stick with that. I think it is best to use the name that the service provider uses. In this case it seems to be "API token": https://circleci.com/docs/managing-api-tokens/